python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -328,8 +333,85 @@ def get_trainer_program(self):
         # FIXME(typhoonzero): Also ops like clip_gradient, lrn_decay?
         delete_ops(self.origin_program.global_block(), self.optimize_ops)
         self.origin_program.__str__()
+
         return self.origin_program
 
+    def _get_trainer_startup_program(self, program=None):
```

Comment:
`program` => `startup_program` should be more clear.

Reply:
Done.